daffodil-runtime2/src/test/resources/org/apache/daffodil/runtime2/ex_nums.dfdl.xsd

```diff
@@ -150,7 +150,7 @@
                                         dfdl:byteOrder="littleEndian"/>
                             <xs:element name="le_uint8" type="xs:unsignedByte"
                                         dfdl:byteOrder="littleEndian"/>
-                            <xs:element name="le_nonNegativeInteger10" type="xs:nonNegativeInteger"
+                            <xs:element name="le_uint10" type="xs:unsignedInt"
                                         dfdl:byteOrder="littleEndian"
                                         dfdl:length="10"
```

Comment:
I also still want ex_nums to test the integer and nonNegativeInteger types, but we'll need new elements for them.  You don't have to do it in this PR - I can add the new elements later.